### ISSUE TITLE
add support for models on any image normalization

### ISSUE THREAD

**Issue by Mayukhdeb** [Mayukhdeb/torch-dreams#44]
For example, this transform converts an imagenet transformed image tensor to `0, 255`, this can be generalized. 

```python
inv = transforms.Compose([ transforms.Normalize(mean = [ 0., 0., 0. ],
                                                     std = [ 1/0.229, 1/0.224, 1/0.225 ]),
                                transforms.Normalize(mean = [ -0.485, -0.456, -0.406 ],
                                                     std = [ 1., 1., 1. ]),
                               ])   

class InverseTransform(nn.Module):
    def __init__(self, inverse_transform):
        super().__init__()
        self.inverse_transform = inverse_transform
        self.scale = 255

    def forward(self, x):
        x=  self.inverse_transform(x)
        x = x * self.scale
        # print(x.max(), x.min(), x.mean())
        return x
```

**Comment by Mayukhdeb** [comment]
We can generalise this as:

```python

dreamer.set_custom_normalization(
    mean = [0.5, 0.5, 0.5], 
    std = [0.5, 0.5, 0.5]
)
```

and then `inv` would be like: 

```python
inv = transforms.Compose([ transforms.Normalize(mean = [ 0., 0., 0. ],
                                                     std = [ 1/std[0], 1/std[1], 1/std[2]),
                                transforms.Normalize(mean = [ -mean[0], -mean[1], -mean[2] ],
                                                     std = [ 1., 1., 1. ]),
                               ])   
```
